app/views/NewAuthYesNoView.scala.html

```diff
@@ -40,10 +40,21 @@
 
         @inputYesNo(
             field = form("value"),
-            yesNoType = OrdinaryYesNo(
+            yesNoType = YesNoWithAdditionalHtml(
                 heading = messages("newAuthYesNo.heading"),
-                caption = Some(messages("site.caption", mrn.toString))
-            )
+                caption = Some(messages("site.caption", mrn.toString)),
+                additionalHtml = Html(
+                        s"""
+                        <p>${messages("newAuthYesNo.paragraph").toParagraph}</p>
+                        <p class = "govuk-hint">${messages("newAuthYesNo.hint")}</p>
+                            <ul class="govuk-list govuk-list--bullet govuk-hint">
+                                  <li>${messages("newAuthYesNo.bullet1")}</li>
+                                  <li>${messages("newAuthYesNo.bullet2")}</li>
+                                  <li>${messages("newAuthYesNo.bullet3")}</li>
+                            </ul>
+                        """
```

Comment:
Why aren't we providing the hint text to the hint param?

Reply:
Something like

```
@(form: Form[Boolean],mrn: MovementReferenceNumber, arrivalId: ArrivalId, mode: Mode)(implicit request: Request[_], messages: Messages)

@html = {
    <p>@messages("newAuthYesNo.hint")</p>
    <ul class="govuk-list govuk-list--bullet govuk-hint">
        <li>@messages("newAuthYesNo.bullet1")</li>
        <li>@messages("newAuthYesNo.bullet2")</li>
        <li>@messages("newAuthYesNo.bullet3")</li>
    </ul>
}

@mainTemplate(
    title = messages("newAuthYesNo.title"),
    showBackLink = true
) {

        @errorSummary(form.errors)

        @formHelper(action = controllers.routes.NewAuthYesNoController.onSubmit(arrivalId, mode), Symbol("autoComplete") -> "off") {

        @inputYesNo(
            field = form("value"),
            yesNoType = YesNoWithAdditionalHtml(
                heading = messages("newAuthYesNo.heading"),
                caption = Some(messages("site.caption", mrn.toString)),
                additionalHtml = messages("newAuthYesNo.paragraph").toParagraph
            ),
            hint = Some(HtmlContent(html))
        )

        @button(attributes = Map("id" -> "submit"))
    }
}
```

Reply:
@jackheslop96 done  please re-review